### PR TITLE
Add some MOTDs

### PR DESCRIPTION
**Pull request by Firesphere**
## The Issue

Randy asked for more MOTDs in Discord

## How This PR Solves The Issue

I added some MOTDs

## Manual Testing Instructions

Ehm... keep restarting a project until the new message arrives?

## Automated Testing Overview

Is this even automatically tested?

## Related Issue Link(s)

https://discord.com/channels/664580571770388500/1068592266752573440/1132010094033375424

## Release/Deployment Notes

I don't believe there are any serious ramifications.

### CODE REVIEW

File: remote-config.jsonc [hunk 1]
```diff
@@ -60,12 +60,18 @@
...
+				    "message": "Need to get the database and files from your Platform.sh stack? Try `ddev pull platform`! https://ddev.readthedocs.io/en/latest/users/providers/platform"
+				},
 				{
 					"message": "Use `ddev composer` instead of `composer` to ensure your project uses the Composer and PHP versions it specifies."
 				},
 				{
 					"message": "`ddev npm` is the right way to run npm commands in your web container."
 				},
+				{
+					"message": "`ddev yarn` is the right way to run yarn commands in your web container."
```
Comment (rfay) [inline remote-config.jsonc]:
```suggestion
					"message": "`ddev yarn` is a great way to run yarn commands in your web container."
```

File: remote-config.jsonc [hunk 2]
```diff
@@ -60,12 +60,18 @@
 				{
 				    "message": "Platform.sh sponsors DDEV by paying its lead maintainer to work on DDEV full time!"
 				},
+				{
+				    "message": "Need to get the database and files from your Platform.sh stack? Try `ddev pull platform`! https://ddev.readthedocs.io/en/latest/users/providers/platform"
```
Comment (rfay) [inline remote-config.jsonc:64]: This is great, thanks.

**Review by rfay — CHANGES_REQUESTED** [review]
Just a suggested change. Wish we had a good way to do a line break. Probably have to figure that out.

File: remote-config.jsonc (hunk 1 above)
Comment (Firesphere) [reply remote-config.jsonc]: I copied this from the NPM command above ("the right way"), for consistency, I'd say either have both "a great way" or both "the right way"?

Comment (rfay) [reply remote-config.jsonc]: "the right way" just seemed kind of prescriptive. Not sure it matters. But using `yarn` inside the container or using `ddev exec yarn`... nothing wrong with any of those.

**Review by rfay — APPROVED** [review]
Thanks!

**Pull request merged**